which-key.el

```diff
@@ -1361,16 +1365,33 @@ the ordering of classes are listed below.
 special (SPC,TAB,...) < single char < mod (C-,M-,...) < other."
   (which-key--key-description< (car acons) (car bcons)))
 
+(defsubst which-key--string-group-descr-face-p (str)
+  (face-equal 'which-key-group-description-face (get-text-property 0 'face str)))
+
 (defsubst which-key-description-order (acons bcons)
   "Order descriptions of A and B.
 Uses `string-lessp' after applying lowercase."
-  (string-lessp (downcase (cdr acons)) (downcase (cdr bcons))))
+  (let ((adesc (nth 2 acons))
+        (bdesc (nth 2 bcons)))
```

Comment:
These are not correct. `acons` and `bcons` should be cons cells. `(nth 2 x)` would be the third element of a list